setup.py

```diff
@@ -29,20 +29,16 @@
 # 'Development Status :: 5 - Production/Stable'
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
-    "grpcio >= 1.47.0, < 2.0dev",  # https://github.com/googleapis/python-bigquery/issues/1262
-    "grpcio >= 1.49.1, < 2.0dev; python_version>='3.11'",
     # NOTE: Maintainers, please do not require google-api-core>=2.x.x
```

Comment:
I think we mentioned this at our last meeting, but it was because grpcio was both in dependencies and bqstorage optional extras, I believe it was supposed to be moved there but was not until the referenced issue above it was closed